### DISCUSSION TITLE
How to pass a custom theme to CustomizationProvider within Flex.setProviders

### DISCUSSION THREAD

**Discussion by WillCKyle** [twilio-labs/paste#2750] [Q&A] [answered]
I am using Flex.setProviders in a typescript plugin as described [here](https://www.twilio.com/docs/flex/developer/ui/overview-of-flex-ui-programmability-options#theming-flex-ui):

`Flex.setProviders({
        PasteThemeProvider: CustomizationProvider
     });`

How can I pass it a custom theme, as I have done here:

`<CustomizationProvider key="CustomizationProvider" baseTheme="dark" theme={pasteTheme} >`

**Comment by gloriliale** [comment, marked as answer]
Hi @WillCKyle,

The Flex team would be the most useful for answering questions about the Flex API. As per https://github.com/twilio-labs/paste/discussions/2742, we'd recommend checking in with someone from the Flex support side, or reaching out to support@twilio.com! Many thanks!
